options/posix/generic/unistd.cpp

```diff
@@ -236,8 +236,8 @@ int faccessat(int dirfd, const char *pathname, int mode, int flags) {
 }
 
 int fchown(int fd, uid_t uid, gid_t gid) {
-	MLIBC_CHECK_OR_ENOSYS(mlibc::sys_fchownat, -1);
```

Comment:
Another possibility would be to try to use `sys_fchownat` and fall back to a `sys_fchown`.